boa_tester/src/main.rs

```diff
@@ -194,41 +194,42 @@ static IGNORED: Lazy<Ignored> = Lazy::new(|| {
 
 /// Boa test262 tester
 #[derive(Debug, Parser)]
-#[clap(author, version, about, name = "Boa test262 tester")]
+#[command(author, version, about, name = "Boa test262 tester")]
 enum Cli {
     /// Run the test suite.
     Run {
         /// Whether to show verbose output.
-        #[clap(short, long, parse(from_occurrences))]
+        #[arg(short, long, action = clap::ArgAction::Count)]
```

Comment:
I'd just import `ArgAction`, since there are already other structs imported and using fully qualified syntax in this case doesn't do much.